dataland-frontend/src/assets/scss/components/form.scss

```diff
@@ -504,7 +504,7 @@ input[type="checkbox"], input[type="radio"]{
     .formkit-options {
       display: flex;
       padding: .5rem 0 0 0;
-      .formkit-option:nth-child(2n) {
+      .formkit-option:nth-child(n+2) {
         margin-left: 10%;
       }
```

Comment:
When editing data, and I don't want to re-use an old report, I can't:
![image](https://user-images.githubusercontent.com/47553592/236821864-166bb8a2-2cf1-4bee-95bd-fc34f8faed4d.png)
But I can't remove this report as it comes from the "edit" functionality

Reply:
Maybe I have misunderstood this comment, but you should definitely be able to remove "old" reports by clicking the respective "X" in its container.
Then it should be gone from the referenced reports.
I just tried it and it worked locally for me.

Reply:
Might this actually rather be related to the edit data form not loading in correctly and having erroneously unfilled fields? An issue that should be fixed with Emanuels latest commit.

Reply:
Whatever I did - now it works. Sorry for bothering.